src/current/_includes/releases/v24.3/v24.3.14.md

```diff
@@ -0,0 +1,84 @@
+## v24.3.14
+
+Release Date: May 28, 2025
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v24-3-14-operational-changes">Operational changes</h3>
+
+- Changed the default value of the cluster setting `admission.l0_file_count_overload_threshold` to `4000`.
+ [#145919][#145919]
+
+<h3 id="v24-3-14-db-console-changes">DB Console changes</h3>
+
+- Schema insights that recommend replacing an index were previously a two-statement command consisting of a `CREATE INDEX` and a `DROP INDEX` statement. When these two DDL statements were run as a single batched command, it was possible for one statement to succeed and one to fail. This is because DDL statements do not have the same atomicity guarantees as other SQL statements in CockroachDB. Index-replacement insights are now a single `CREATE INDEX` statement followed by a comment with additional DDL statements to be run manually: an `ALTER INDEX ... NOT VISIBLE` statement, which makes the old index invisible to the optimizer, followed by a `DROP INDEX` statement that should only be run after making the old index invisible and verifying that workload performance is satisfactory.
+ [#145988][#145988]
+
+<h3 id="v24-3-14-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug where using values `changefeed.aggregator.flush_jitter` and `min_checkpoint_frequency` such that `changefeed.aggregator.flush_jitter * min_checkpoint_frequency < 1` would cause a panic. Jitter will now be disabled in this case.
+ [#144425][#144425]
+- Fixed a bug in the DB Console where the **Drop unused index** tag appeared multiple times for an index on the **Indexes** tab of the table details page.
+ [#144652][#144652]
+- Fixed the following bugs in the **Schedules** page of the DB Console:
+- Fixed a bug where the **Schedules** page displayed only a subset of a cluster's schedules. The **Schedules** page now correctly displays all schedules.
+- Fixed a bug where manually updating the `show` or `status` parameters in the URL (e.g., `http://127.0.0.1:8080/#/schedules?status=ACTIVE&show=50`) caused the **Schedules** page to fail to load.
+ [#144805][#144805]
```

Comment:
I think you need to indent the sub-bullets here, because they're rendering as separate notes.